README.rst

```diff
@@ -1106,6 +1106,21 @@ Below is an example of publishing to Vault (using token auth with a local dev in
    example_number    42
    example_string    bar
 
+Publishing to Azure Key Vault
+*******************
```

Comment:
For this to be valid RST, you need to use (at least) the same amount of `*` characters than the above line has letters and spaces.
```suggestion
*****************************
```

Reply:
Done